source/mainnet/net/resources/release-notes-mainnet.rst

```diff
@@ -12,6 +12,14 @@ Mainnet 4: Sirius
 
    Prior to Sirius, the nodes enforced that a transaction could not be deployed until 2 hours before its expiry date. With Sirius, node validation of transactions has been improved and the 2 hour window has been removed.
 
+October 5, 2022
+
+Concordium Client 4.2.0
+-----------------------
+
+- Fix handling of ``--no-confirm`` in ``contract init``, ``contract update``, ``module deploy``, and ``register data`` transactions. This flag is now respected.
+- Add support for import of keys from |bw|.
```

Comment:
Did you deliberately omit the 
```
Make the output of CCD amounts consistent when printing transaction outcomes. The output should now always x.y CCD for some x and y.
```
?

Maybe we could instead add something like "Fix some inconsistencies in the display format of CCD amounts."?

Reply:
It wasn't in the changelog when I took a copy and made the pull request. So no, it was not deliberately omitted. I have added it now.